tpm2/keydata.go

```diff
@@ -201,3 +204,66 @@ func (k *sealedKeyDataBase) validateData(tpm *tpm2.TPMContext, session tpm2.Sess
 
 	return pcrPolicyCounterPub, nil
 }
+
+// SealedKeyData corresponds to a sealed key data object created by
+// [ProtectKeyWithTPM], [ProtectKeysWithTPM], or [ProtectKeyWithExternalStorageKey].
```

Comment:
nitpick: we weren't using \[\] (consistently) before?

Reply:
I've not used that consistently before because I wasn't aware of it - I've started using it in recent commits because it's nice to have the local doc links.